drlhp/deprecated/a2c/a2c.py

```diff
@@ -3,17 +3,20 @@
 import queue
 import time
 
+from copy import deepcopy
```

Comment:
There's a bunch of debugging stuff in this file; I'm going to ignore it since the entire file since it's deprecated and looks to be in the middle of being fixed.

Reply:
Yeah, at this point I'm not even sure it makes sense to keep the deprecated files around rather than just blowing them away; in theory it might be nice to have them, but you can also find the code in Matthew's version of the repo? Curious if you have thoughts on this.